src/main/openapi/discovery.yaml

```diff
@@ -351,6 +373,20 @@ components:
       required:
         - target
         - relation
+    DatasetOrganization:
+      properties:
+        name:
+          type: string
+          title: Name of organization
+        title:
+          type: string
+          title: Title of organization
+        description:
+          type: string
+          title: Description of organization
+        image_url:
```

Comment:
this is an url pointing to ckan? if this is the case, we cannot hide ckan behind twingate as we were thinking of

Reply:
important: I suggest we bring this discussion to another story, as the network layer must be transparent to the application layer.

Reply:
not sure, but I would say yes. Probably like media link on ckan. Then I guess we cannot have image on organization